src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -95,20 +106,71 @@ export class MarkdownToolbarComponent {
 			this.optionParagraph,
 			undefined
 		);
-		dropdownMenuActionViewItem.render(buttonDropdownContainer);
+		dropdownMenuActionViewItem.render(this.buttonDropdownContainer);
 		dropdownMenuActionViewItem.setActionContext(this);
 
+		// Hide link and image buttons in WYSIWYG mode
+		if (this.cellModel.showPreview && !this.cellModel.showMarkdown) {
```

Comment:
I'm worried about duplication here.

How can we bring this # down of declaring these arrays?